docs/user/contacts.md

```diff
@@ -0,0 +1,14 @@
+# Contact Us
+
+Welcome to the Vald project!!
+
+Vald is an open source project and we would love to hear from you.
+
+If you are interested in Vald, or you have any question about Vald, please feel free to contact us.
+
+We recommend to communicate on slack, but using E-mail is also welcome.
+ 
+Slack: [Vald Community](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) 
+E-mail: vald@vdaas.org
```

Comment:
```suggestion

E-mail: vald@vdaas.org
```

Reply:
I think this line is unnessary.

Reply:
Since the contents have the same meaning, need not to put a blank line. 
What do you think?

Reply:
I think it is easier to read, but it depends on person. Lets here kiichiro-san's comment later :)

Reply:
sorry. 
I preview my markdown.
It was like below.
```
Slack: Vald Community E-mail: vald@vdaas.org
```

I think your option is good.
sorry, I will fix it soon

Reply:
```suggestion
E-mail: [vald@vdaas.org](mailto:vald@vdaas.org)
```